### ISSUE TITLE
API Keys not read from config.ini

### ISSUE THREAD

**Issue by capacitor0** [OWASP/Amass#192]
Hello, I have configured the config.ini file like that : 
```
[…]
# Provide API key information for a data source
#[AlienVault]
#apikey =

[BinaryEdge]
apikey = xxxxxxxx-b987-446a-997d-3cbc9003b693

[Censys]
apikey = xxxxxxxx-3f1e-4261-bec7-f7754c5746c4
secret = xxxxxxxxXQn5h0Yrz8ioqVWxpWQwvx9Z

#[CertDB]
#username =
#password =

#[CIRCL]
#username =
#password =

#[DNSDB]
#apikey =

[PassiveTotal]
username = xxxxxxxx
apikey = xxxxxxxx1e7b9410a3440f59d1e86ae40cec1b2c8b6cb3e0791a8c6863d2d81c

[SecurityTrails]
apikey = xxxxxxxxs0TdX6BuwKHfGdY2NmHW81Fc

#[Shodan]
#apikey =

# Provide your Twitter App Consumer API key and Consumer API secret key
#[Twitter]
#apikey =
#secret =

# The apikey must be an API access token created through the Investigate management UI
#[Umbrella]
#apikey =

# URLScan can be used without an API key, but the key allows new submissions to be made
[URLScan]
apikey = xxxxxxxx-d8fa-4b18-9e33-24dec51eb644

[VirusTotal]
apikey = xxxxxxxxb7d0a2b8798d3c661a2924406e6e44d8da654a453a870f4642e21bbe
```
But when checking the log file I've got this error : 
```
14:30:09.613568 AlienVault: API key data was not provided
14:30:09.614736 BinaryEdge: API key data was not provided
14:30:09.614937 Censys: API key data was not provided
14:30:09.615029 CertDB: API key data was not provided
14:30:09.615452 CIRCL: API key data was not provided
14:30:09.786886 DNSDB: API key data was not provided
14:30:09.787095 PassiveTotal: API key data was not provided
14:30:09.787226 SecurityTrails: API key data was not provided
14:30:14.230684 Shodan: API key data was not provided
14:30:14.230788 Twitter: API key data was not provided
14:30:14.230857 Umbrella: API key data was not provided
14:30:14.230879 URLScan: API key data was not provided
14:30:14.230931 VirusTotal: API key data was not provided
```
So could you please help me?

**Comment by caffix** [comment]
How did you point Amass to your configuration file?

**Comment by capacitor0** [comment]
I run the following command :
`amass enum -config config.ini -d some.domain`

**Comment by caffix** [comment]
Did you install Amass using the Snapcraft package?

**Comment by capacitor0** [comment]
I used brew, (https://github.com/OWASP/Amass/blob/master/doc/install.md#homebrew-macos)

**Comment by electroxero** [comment]
Same problem here, i have tried a command like this:

**Comment by JonathanJonathanJonathan** [comment]
Also the same problem here. Tried different ways to load the config.ini, none of them work (relative path, absolute path, default dir). The problem is that the API keys are not properly detected, other config properties work.

**Comment by caffix** [comment]
We found the cause of this issue and fixed it in v3.0.22. Thank you for helping us hunt this down!
